torch/distributed/pipeline/sync/microbatch.py

```diff
@@ -132,39 +135,32 @@ def _setitem_by_slice(self, index: slice, value: Tensors) -> None:
         self.value = value[0]
 
 
-def check(input: TensorOrTensors) -> None:
+def check(*inputs) -> None:
```

Comment:
nit: no type hint?

Reply:
The PRs on top allow for generic inputs, as a result I didn't put a type hint. This is similar to *args.